images/collector.Dockerfile

```
@@ -1,15 +1,46 @@
-FROM cimg/go:1.16
-
-USER 0
-
-RUN apt-get update && \
-  apt-get upgrade && \
-  apt-get install -y --no-install-recommends \
-    lsb-release \
-    cmake \
-    python3-distutils \
-    clang-format \
-    patch && \
+FROM quay.io/centos/centos:stream8
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+### cci-export support (and google-cloud-sdk repo)
+COPY ./static-contents/ /static-tmp
+RUN set -ex \
+  && find /static-tmp -type f -print0 | \
+    xargs -0 -I '{}' -n1 bash -c 'dir="$(dirname "${1}")"; new_dir="${dir#/static-tmp}"; mkdir -p "${new_dir}"; cp "${1}" "${new_dir}";' -- {} \
+  && rm -r /static-tmp
+
+RUN yum update -y && \
+    yum install -y epel-release dnf-plugins-core && \
+    yum config-manager --set-enabled powertools && \
+    yum -y groupinstall "Development Tools" && \
+    yum install -y \
+        clang-tools-extra \
+        cmake \
+        google-cloud-sdk \
+        jq \
+        python3 \
```

Looks like this installs python 3.6 and it is breaking one of our CI steps: https://app.circleci.com/pipelines/github/stackrox/collector/9080/workflows/685cfead-ff10-448b-9514-96b16a6adf42/jobs/253935

```
protobuf requires Python '>=3.7' but the running Python is 3.6.8
```

Think we can directly change it to install python 3.8 with this
```suggestion
        python38 \
```